misc/scratch/remote-dev.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "vs code remote dev",
+  "launch_script": "true",
+  "instance_type": "c6a.4xlarge",
```

Comment:
This is the cheapest amd64 16 vcpu box on ec2. 8 cores was just not enough when I was using the default dev-box as a daily driver.